permission_handler_platform_interface/lib/src/permission_status.dart

```diff
@@ -59,7 +64,8 @@ extension PermissionStatusValue on PermissionStatus {
 
 /// Utility getter extensions for the [PermissionStatus] type.
 extension PermissionStatusGetters on PermissionStatus {
-  /// If the user denied access to the requested feature.
+  /// If the user denied access to the requested feature or
+  /// permission needs to be asked first.
```

Comment:
The word `or` should be replaced by a comma:

```suggestion
  /// If the user denied access to the requested feature,
  /// permission needs to be asked first.
```